onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -8,7 +8,7 @@
 namespace onnxruntime {
 namespace test {
 TEST(ResizeOpTest, ResizeOpLinearDownSampleTest_tf_crop_and_resize) {
-  OpTester test("Resize", 11);
+  OpTester test("Resize", 13);
```

Comment:
bump almost all test to opset-13 (safe to do this and will keep the earlier behavior covered too)

Reply:
Unrelated to this PR, but should OpTester take a list of opsets instead of just one? That way we could do something like OpTester test("Resize", {11, 13}). 

---
In reply to: [510535987](https://github.com/microsoft/onnxruntime/pull/5575#discussion_r510535987) [](ancestors = 510535987)

Reply:
This is a good idea but requires some work in `provider_test_utils` (support a vector of opsets and basically run the current logic in a loop for each opset in the vector) that may not be trivial enough to include and leverage in this change. Perhaps we can address this enhancement in a different change and cover all the ops that can leverage that ?